plone/app/blocks/layoutbehavior.py

```diff
@@ -244,17 +256,13 @@ def sectionSiteLayout(self, value):
     def tile_layout(self):
         return self.content or u''
 
+    def content_layout_path(self):
+        path = self.contentLayout
+        return path or super(LayoutAwareBehavior, self).content_layout_path()
+
     def content_layout(self):
-        if self.contentLayout:
-            try:
-                path = self.contentLayout
-                resolved = resolveResource(path)
-                return applyTilePersistent(path, resolved)
-            except (NotFound, RuntimeError, IOError):
-                pass
-        elif self.customContentLayout:
+        if self.customContentLayout and not self.contentLayout:
```

Comment:
This is indeed confusing, but was that way and also the tests are checking it in the current order.

I would expect it to check for custom layout, then for a predefined selected (by path) and at the end for the default. 

But to change this is outside of this PR.